### PR TITLE
[DOCS] Add instructions to turn off Endpoint diagnostic data

### LINKED ISSUE #1591: [DOCS] Instructions to turn off Endpoint alert diagnostics in the Endpoint policy
## Description 

We should add documentation to inform users how to turn off Endpoint alert diagnostics in the app.  This is done via the Endpoint policy and it may make sense to add it in this section: https://www.elastic.co/guide/en/security/8.0/configure-endpoint-integration-policy.html#adv-policy-settings

## Acceptance Test Criteria

- [ ] First make a note that if users' goal is to stop us from collecting telemetry, including diagnostic alerts, they can do so by turning off Kibana telemetry: https://www.elastic.co/guide/en/kibana/8.0/telemetry-settings-kbn.html

- [ ] If the users' goal is to closely manage everything the Endpoint streams to their own clusters, then they can stop the Endpoint from streaming diagnostic alerts as below.

- [ ] On Endpoints where you want to disable diagnostic alerts, go to the Policy for those Endpoints and access the Advanced section.

image

- [ ] Set these fields to "False" and save

image

- [ ] This will stop the Endpoint from streaming diagnostic alerts to a hidden index on the user's own cluster.
 
## Notes

- Add all appropriate labels to the issue, especially the version number label. 
- Be sure to add any necessary screenshots, code text or console commands for clarity. 
- Include any conditions or caveats that may affect customers.

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1591.

The **Configure the Endpoint Security integration policy** page is already pretty long, so the procedure for turning off diagnostics is on its own page.

Previews:
* [Turn off Endpoint Security diagnostic data](https://security-docs_1641.docs-preview.app.elstc.co/guide/en/security/master/endpoint-diagnostic-data.html) - New page for procedure
* [Configure the Endpoint Security integration policy](https://security-docs_1641.docs-preview.app.elstc.co/guide/en/security/master/configure-endpoint-integration-policy.html#adv-policy-settings) - Link from integration policy configuration page

### CODE REVIEW

**Review by kevinlog — COMMENTED** [review]
this looks good to me.

Would want @joe-desimone and @stevewritescode to take a look too since this is insight into the inner workings of telemetry

**Review by joe-desimone — APPROVED** [review]
Looks good to me

File: docs/getting-started/endpoint-diagnostic-data.asciidoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[[endpoint-diagnostic-data]]
+= Turn off {endpoint-sec} diagnostic data
+
+By default, {endpoint-sec} streams diagnostic data to your cluster, which Elastic uses to tune protection features. You can stop producing this diagnostic data by configuring the advanced settings in the {endpoint-sec} integration policy.
+
+NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control Kibana telemetry in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
```
Comment (nastasha-solomon) [inline docs/getting-started/endpoint-diagnostic-data.asciidoc:6]: Are we officially calling the telemetry feature "Kibana telemetry"? I thought it was just "telemetry".

File: docs/getting-started/endpoint-diagnostic-data.asciidoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+. In the {security-app}, go to *Manage* -> *Endpoints* to view the Endpoints list.
+. On endpoints where you want to disable diagnostic data, click the integration policy in the *Policy* column.
```
Comment (nastasha-solomon) [inline docs/getting-started/endpoint-diagnostic-data.asciidoc:9]:
```suggestion
. Go to the endpoint that you want to disable diagnostic data on, then click the integration policy in the *Policy* column.
```

Comment (nastasha-solomon) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:9]: Mainly leaving this to indicate that endpoint should be singular.

**Review by nastasha-solomon — APPROVED** [review]
Just two minor comments. Overall, LGTM!

File: docs/getting-started/configure-integration-policy.asciidoc [hunk 3]
```diff
@@ -123,6 +123,8 @@ image::images/register-as-antivirus.png[Detail of Register as antivirus option.]
 
 Users with unique configuration and security requirements can select **Show Advanced Settings** to configure the policy to support advanced use cases. Hover over each setting to view its description.
 
+In this section, you can <<endpoint-diagnostic-data, turn off Endpoint Security diagnostic data>>.
```
Comment (jmikell821) [inline docs/getting-started/configure-integration-policy.asciidoc:126]: Very small nit here. 

```suggestion
In this section, you can <<endpoint-diagnostic-data, turn off {endpoint-sec} diagnostic data>>.
```

File: docs/getting-started/endpoint-diagnostic-data.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:6]: @nastasha-solomon This wasn't intended as an official name of the feature, just a short description. Maybe "telemetry in Kibana"?
```suggestion
NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control telemetry in Kibana in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
```

File: docs/getting-started/endpoint-diagnostic-data.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:9]: @nastasha-solomon Gotcha, I think we can stay close to the original and still revise to singular:
```suggestion
. On the endpoint where you want to disable diagnostic data, click the integration policy in the *Policy* column.
```

File: docs/getting-started/endpoint-diagnostic-data.asciidoc (hunk 1 above)
Comment (jmikell821) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:6]: I _think_ it's just `telemetry` too.

File: docs/getting-started/endpoint-diagnostic-data.asciidoc (hunk 2 above)
Comment (jmikell821) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:9]: Revision to @nastasha-solomon suggestion: 

> Go to the endpoint for which you want to disable diagnostic data, then click the integration policy in the **Policy** column.

Can also use "locate the endpoint" in lieu of "go to".

**Review by jmikell821 — COMMENTED** [review]
Just a few small nits. Thanks!

File: docs/getting-started/endpoint-diagnostic-data.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/getting-started/endpoint-diagnostic-data.asciidoc:6]: Revised the suggestion for just "telemetry"

**Review by benironside — APPROVED** [review]

**Pull request merged**